### ISSUE TITLE
UnboundLocalError: local variable 'outpath' referenced before assignment

### ISSUE THREAD

**Issue by aliabbasjp** [interrogator/corpkit#41]
Now I get following error: on command : parse corpus

```

ParserAnnotator: 173.3 sec.
NERCombinerAnnotator: 14.6 sec.
DeterministicCorefAnnotator: 8.2 sec.  
TOTAL: 197.9 sec. for 45060 tokens at 227.7 tokens/sec.
Pipeline setup: 6.7 sec.
Total time for StanfordCoreNLP pipeline: 211.0 sec.
14:34:22: Parsing finished. Moving parsed files into place ...
Traceback (most recent call last):
  File "/home/domas/anaconda2/lib/python2.7/site-packages/corpkit/env.py", line 2168, in interpreter
    out = run_command(tokens)
  File "/home/domas/anaconda2/lib/python2.7/site-packages/corpkit/env.py", line 1113, in run_command
    out = command(tokens[1:])
  File "/home/domas/anaconda2/lib/python2.7/site-packages/corpkit/env.py", line 1437, in parse_corpus
    parsed = to_parse.parse(**kwargs) 
  File "/home/domas/anaconda2/lib/python2.7/site-packages/corpkit/corpus.py", line 930, in parse
    **kwargs
  File "/home/domas/anaconda2/lib/python2.7/site-packages/corpkit/make.py", line 388, in make_corpus
    rename_all_files(outpath)
UnboundLocalError: local variable 'outpath' referenced before assignment
```

**Comment by interrogator** [comment]
Oh, my mistake. OK, it's fixed in the master branch on GitHub now, so update your copy and try again. Can you confirm that the `parse corpus` command works for you now?

**Comment by interrogator** [comment]
Apparently fixed by eabb2d338a82899e8127c624e5ff997264db771c.
